### ISSUE TITLE
Webpack 5 support

### ISSUE THREAD

**Issue by daniele-orlando** [retyui/clean-css-loader#15]
Please update the `package.json` peer deps to support **WebPack 5**.

```
npm ERR! Found: webpack@5.16.0
npm ERR! Could not resolve dependency:
npm ERR! peer webpack@"^1.0.0 || ^2.0.0 || ^3.0.0 || ^4.0.0" from clean-css-loader@2.0.0
npm ERR! node_modules/clean-css-loader
npm ERR!   clean-css-loader@"2.0.0" from the root project
```

**Comment by retyui** [comment]
Fixed in [3.0.0](https://github.com/retyui/clean-css-loader/blob/master/CHANGELOG.md#300)

- diff [2.0.0...3.0.0](https://npmfs.com/compare/clean-css-loader/2.0.0/3.0.0/)

**Comment by daniele-orlando** [comment]
Thank you very much. ❤️
